packages/odyssey-storybook/src/components/odyssey-mui/Button/Button.stories.tsx

```diff
@@ -63,77 +57,154 @@ const storybookMeta: Meta<ButtonProps> = {
+  args: {
+    text: "Add crew",
+    size: "medium",
+  },
+  play: async ({ args, canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = await canvas.getByText("Add crew");
+    await userEvent.hover(button);
+    await userEvent.click(button);
+    await expect(args.onClick).toHaveBeenCalledTimes(1);
+  },
 };
 
-export const ButtonLarge = Template.bind({});
-ButtonLarge.args = {
-  text: "Add crew",
-  size: "large",
+export const ButtonLarge: StoryObj<ButtonProps> = {
+  args: {
+    text: "Add crew",
+    size: "large",
+  },
+  play: async ({ args, canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = await canvas.getByText("Add crew");
+    await userEvent.hover(button);
+    await userEvent.click(button);
+    await expect(args.onClick).toHaveBeenCalledTimes(1);
+  },
 };
 
-export const ButtonFullWidth = Template.bind({});
-ButtonFullWidth.args = {
-  text: "Add crew",
-  isFullWidth: true,
+export const ButtonFullWidth: StoryObj<ButtonProps> = {
+  args: {
+    text: "Add crew",
+    isFullWidth: true,
+  },
+  play: async ({ args, canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = await canvas.getByText("Add crew");
+    await userEvent.hover(button);
+    await userEvent.click(button);
+    await expect(args.onClick).toHaveBeenCalledTimes(1);
+  },
 };
 
-export const ButtonPrimaryDisabled = Template.bind({});
-ButtonPrimaryDisabled.args = {
-  text: "Add crew",
-  isDisabled: true,
+export const ButtonPrimaryDisabled: StoryObj<ButtonProps> = {
+  args: {
+    text: "Add crew",
+    isDisabled: true,
+  },
+  play: async ({ args, canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = await canvas.getByText("Add crew");
+    await userEvent.hover(button);
+    await userEvent.click(button);
+    await expect(args.onClick).toHaveBeenCalledTimes(0);
+  },
 };
 
-export const ButtonWithIcon = Template.bind({});
-ButtonWithIcon.args = {
-  text: "Add crew",
-  startIcon: <AddIcon />,
+export const ButtonWithIcon: StoryObj<ButtonProps> = {
+  args: {
+    text: "Add crew",
+    startIcon: <AddIcon />,
+  },
+  play: async ({ args, canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = await canvas.getByText("Add crew");
+    await userEvent.hover(button);
+    await userEvent.click(button);
+    await expect(args.onClick).toHaveBeenCalledTimes(1);
+  },
 };
 
-export const IconOnly = Template.bind({});
-IconOnly.args = {
-  startIcon: <AddIcon />,
-  text: undefined,
-  tooltipText: "Add crew",
+export const IconOnly: StoryObj<ButtonProps> = {
+  args: {
+    startIcon: <AddIcon />,
+    text: undefined, // FIXME
+    tooltipText: "Add crew", // FIXME
+  },
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    const button = await canvas.getByRole("button");
+    await expect(button).toHaveAccessibleName();
+  },
```

Comment:
Did you mean to put these `play` options in this PR? It's fine if they're here.

Reply:
Missed this. I guess they carried over from the base branch `storybook-tests`. Strictly speaking I shouldn't have, but I don't think it's worth cleaning right now since they're all due to merge together